docs/src/api/class-browsercontext.md

```diff
@@ -219,6 +225,7 @@ Whether to pass the argument as a handle, instead of passing by value. When pass
 supported. When passing by value, multiple arguments are supported.
 
 ## async method: BrowserContext.exposeFunction
+**DEPRECATED** Use [`method: BrowserContext.exposeBinding`] instead.
```

Comment:
lets remove it maybe from Python directly?

Reply:
We actually considered it and decided to leave it as is. I'll remove the deprecation note.